content/en/docs/code/cri-api-dev-policies.md

```diff
@@ -0,0 +1,126 @@
+API needed for a Kubernetes feature before this Kubernetes feature can be
+promoted to GA.
+
+## Safe Kubernetes defaults
+
+The feature cannot be enabled by default in Kubernetes as a beta feature before
+the required APIs are implemented in both container runtimes (containerd and
+CRI-O) and there is a GA release of a container runtime for each. The feature
+can be marked as beta, but disabled by default, if there is only one container
+runtime implementation of a required API that is released as GA. Note, as for
+any Kubernetes features, at least one release with the beta feature enabled by
+default is required before it is progressing to GA.
+
+## Guarantee portability
+
+The feature design (KEP PR) MUST be lgtm-ed by container runtime maintainers of
+CRI-O and containerd.
+
+The feature can only be merged as alpha in Kubernetes, if there is an agreement
+from both container runtime maintainers on the feature design in general and API
+shape. Gaining this agreement will often involve authoring the pull request
+demonstrating an API implementation to the both container runtime repositories
+or an alternative way for container runtime maintainers to confirm viability of
+suggested CRI APIs.
+
+## Guaranteed implementation
+
+CRI API can only be merged if there is a PR in both - Kubernetes repository and
+container runtime repository (at least one) utilizing this API and demoing the
+feature working end to end. See CRI API
+[feature-development](/docs/code/cri-api-version-skew-policy#feature-development)
+documentation.
+
+## Features discoverability
+
+Kubernetes features that depend on the environment or special container runtime
+capabilities **must** have its own explicit API configuration (like Pod API or Node
+API) and **must** not depend on the cluster or node configuration that is not
+clearly exposed via these APIs. For example, it is OK to have Windows specific
+features that are configured via Pod API. But it is not OK to design a feature
+that will work on one container runtime and incompatible with the other
+container runtime. There are three exceptions to this case:
+
+- there will be a different behavior during the feature adoption period while
+  older runtime versions do not support the API yet. In those cases, attempting
+  to try the feature **must** result in failing as fast as possible.  
+- LTS and older versions of container runtimes may not have an implementation of
+  an API and still be widely used by Kubernetes end users.  
+- If any of container runtime underlying systems cannot support the feature
+  in-principle (e.g. [kata containers](https://katacontainers.io/) with CRI-O
+  may have limitations), while CRI-O still supports the feature without these
+  systems configured, this **must** be designed as part of a normal operation.
+  In this case, Pod or Node APIs **must** handle these cases gracefully and
+  those **must** be documented clearly.
+
```

Comment:
Do we want to add general development criteria for the CRI? For example:

- Documenting intended behavior directly in the protocol definition to simplify runtime adoption.
- Simplify the intended change as much as possible.
- Reusing existing fields only if their logical meaning allows it and does not infer with existing features.

Reply:
yes, those likely belong to other document. Added a section "Designing new CRI APIs" to the version skew policy document